### PR TITLE
qqq

### PR DESCRIPTION
**Pull request by testaccount120**
.

### CODE REVIEW

**Comment by testaccount120** [comment]
```java
public static void main(String[]args){
        
        }
```

**Comment by testaccount120** [comment]
first

second

third

**Comment by testaccount120** [comment]
first l
second l
third

File: src/main/java/pack/Class0.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package main.java.pack;
 
 public class Class0 {
-    //chanfberberb
+    //chanfberberbfgmfgfgngn
```
Comment (nika-qa) [inline src/main/java/pack/Class0.java:4]: rgeregreger

**Comment by nika-qa** [comment]
vdsdvsdvsdv

File: src/main/java/Class002.java [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
                 "\n" +
-                "If <upstream> is not specified, the upstream configured in branch.<name>.remote and branch.<name>.merge options will be used (see git-config[1] for details) and the --fork-point option is assumed. If you are currently not on any branch or if the current branch does not have a configured upstream, the rebase will abort.");
+                "If <upstream> evewvwevwevis not specified, the upstream configured in branch.<name>.remote and branch.<name>.merge options will be used (see git-config[1] for details) and the --fork-point option is assumed. If you are currently not on any branch or if the current branch does not have a configured upstream, the rebase will abort.");
 
         System.out.println("All changes made by commits in the current branch but that are not in <upstream> are saved to a temporary area. This is the same set of commits that would be shown by git log <upstream>..HEAD; or by git log 'fork_point'..HEAD, if --fork-point is active (see the description on --fork-point below); or by git log HEAD, if the --root option is specified.\n" +
```
Comment (veronikatimkina) [inline src/main/java/Class002.java:10]: gucuyu

File: src/main/java/Class002.java [hunk 3]
```diff
@@ -3,11 +3,10 @@
 public class Class002 {
     public static void main(String[] args) {
         System.out.println("Iggg <branch> is specifiederherherh, git rebase will perform an automatic git switch <branch> before doing anything else. Otherwise it remains on the current branch.\n" +
-                "\n" +
-                "If <upstream> is not specified, the upstream configured in branch.<name>.remote and branch.<name>.merge options will be used (see git-config[1] for details) and the --fork-point option is assumed. If you are currently not on any branch or if the current branch does not have a configured upstream, the rebase will abort.");
```
Comment (veronikatimkina) [inline src/main/java/Class002.java:7]: 34t34t34t34t

Comment (veronikatimkina) [inline src/main/java/Class002.java:7]: 34t34t34t3

**Comment by veronikatimkina** [comment]
ergergerg

**Pull request closed without merging**